h2o-r/h2o-package/R/connection.R

```diff
@@ -8,6 +8,8 @@
 #'
 #' Once connected, the method checks to see if the local H2O R package version matches the version of H2O running on the server. If there is a mismatch and the user indicates she wishes to upgrade, it will remove the local H2O R package and download/install the H2O R package from the server.
 #'
+#' Arguments \code{ip} and \code{port} can be configured by environment variables \code{H2O.IP} and \code{H2O.PORT}.
```

Comment:
Hi Jan, thx for PR - we are trying to keep both R/Python clients in sync. Could you make the same change in Python client?